src/main/plugin/iso19139.ca.HNAP/templates/hnaplevel1-french.xml

```diff
@@ -337,7 +355,7 @@
                   <gco:CharacterString/>
                   <gmd:PT_FreeText>
                      <gmd:textGroup>
-                        <gmd:LocalisedCharacterString locale="#eng"/>
+                        <gmd:LocalisedCharacterString locale=""/>
```

Comment:
Another case where the locale id was lost

Reply:
I do not know what to make of that; if the process is losing these values then it will happen the first time someone makes content with the template, edits, and then saves?

I will fix these but expect we may have another issue to deal with...